packages/botonic-react/src/components/button.tsx

```diff
@@ -187,43 +197,62 @@ export const Button = props => {
     if (props.webview) {
       return (
         <button
+          // @ts-ignore
+          // eslint-disable-next-line react/no-unknown-property
           url={generateWebviewUrlWithParams(props.webview, props.params)}
           {...disabledProps}
         >
           {props.children}
         </button>
       )
-    } else if (props.path) {
+    }
+
+    if (props.path) {
       const payload = `__PATH_PAYLOAD__${props.path}`
       return (
+        // @ts-ignore
+        // eslint-disable-next-line react/no-unknown-property
```

Comment:
Nit: since you are adding this rule disable in multiple lines, you could add it on top of the file as `/* eslint-disable react/no-unknown-property */`

Reply:
Yes, I know, but I only want it for these lines